helpers/MailHelper.php

```diff
@@ -45,4 +47,42 @@ public static function resetPassword($user)
 
         return self::sendMail($message);
     }
+
+    /**
+     * When admin wants to invite user this method will be called
+     *
+     * @param Invitation $invitation
+     * @return bool
+     */
+    public static function invitation(Invitation $invitation)
```

Comment:
@skavt I think better naming would be `sendInvitation` similar to method below `acceptInvitation`